### ISSUE TITLE
Traceback (most recent call last):

### ISSUE THREAD

**Issue by Amjadf** [Gstayton/WABot#1]
python run.py

Traceback (most recent call last):
  File "run.py", line 3, in <module>
    from layer import EchoLayer
  File "/home/ubuntu/WABot-master/layer.py", line 1, in <module>
    from yowsup.layers.interface                           import YowInterfaceLayer, ProtocolEntityCallback
ImportError: No module named yowsup.layers.interface

**Comment by Gstayton** [comment]
I haven't gotten around to putting together a setup script yet, so the project currently does not have a programmatic way of pulling down dependencies, however, as listed under the _Required Modules_ section of the (albeit short) readme file, it lists both requests and yowsup2 as requirements.

That error in particular is from the fact you're missing yowsup2.

With that said, this project is hardly in a 'feature complete' state, and without enough Python knowledge to understand the project, won't really do much. Also missing is documentation on the required 'credentials.py' file, which is where the bots login details are stored.

Long story short, this bot isn't anywhere near a 'download and run' state. I just have it up on source control for my own use, or if someone wants to help out on the project.
